recipes/kubernetes-helm/build.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+make_goroot_read_only()
+{
+    find $GOROOT -type d -exec chmod 555 {} \;
```

Comment:
Actual question. Why do we want this to be read only?

Reply:
This is related to conda-forge/go-feedstock#14. The kubernetes build scripts where compiling the GO stdlib on the fly, the results ended up in $GOROOT, i.e. $PREFIX/go. Conda-build did not know any better and was packaging those results in the kubernetes package as well.

Setting the goroot directory to read-only prevents the kubernetes build scripts from doing that.

Reply:
Ah ok. Makes sense.

Wonder if we should add this to `go` proper perhaps as part of the activation scripts? We can also set it up so that it is only enabled when `conda-build` is running.